common/test/acceptance/pages/studio/overview.py

```diff
@@ -635,6 +636,75 @@ def proctoring_items_are_displayed(self):
 
         return True
 
+    def select_access_tab(self):
+        """
+        Select the access settings tab.
+        """
+        self.q(css=".settings-tab-button[data-tab='access']").first.click()
+        self.wait_for_element_visibility('#is_prereq', 'Gating settings fields are present.')
+
+    def make_gating_prerequisite(self):
+        """
+        Makes a subsection a gating prerequisite.
+        """
+        if not self.q(css="#is_prereq")[0].is_selected():
+            self.q(css='label[for="is_prereq"]').click()
+        self.q(css=".action-save").first.click()
+        self.wait_for_ajax()
```

Comment:
We have found that wait_for_ajax is problematic-- relying on it often results in a flaky test. If there is a specific thing on the page that you can wait for to signify that the operation is complete, that is much more reliable.

I recommend that you run your new tests many tests with the flaky decorator (min_pass of 20 or 30) before submitting them to ensure they are robust. Remember to remove the flaky decorator before submission though!